xml/Menu/membershipextras.xml

```diff
@@ -6,12 +6,24 @@
     <title>Payment Plan Settings</title>
     <access_arguments>administer CiviCRM</access_arguments>
   </item>
+  <item>
+    <path>civicrm/admin/member/automated-upgrade-rules</path>
+    <page_callback>CRM_MembershipExtras_Page_AutomatedMembershipUpgradeRule</page_callback>
+    <title>Membership automated upgrade rules</title>
```

Comment:
@omarabuhussein  just quick one, I spot in on the screenshot,  should title / menu be "Membership Automated Upgrade Rules" (wording start with capital)

Reply:
ah! thanks @erawat

Reply:
@omarabuhussein  did you update this yet?